nuscr.opam

```diff
@@ -0,0 +1,40 @@
+# This file is generated by dune, edit dune-project instead
```

Comment:
Jane street does so.

Also `dune` has a problem with ignored `.opam` files, because the first step of the build is
`dune subst`, which requires a `.opam` file to begin with, but dune only looks for files under VCS, so it reports the error.

Reply:
ok, you had me at Jane Street does so. They are the cool kidz after all.